client/src/Components/Calculator.jsx

```diff
@@ -0,0 +1,91 @@
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+import LeftPanel from "./LeftPanel";
+import GraphArea from "./GraphArea";
+import ErrorPage from "./ErrorPage";
+
+function Calculator() {
+  const [monthlyInvestment, setMonthlyInvestment] = useState(500);
+  const [investmentPeriod, setInvestmentPeriod] = useState(1);
+  const [rateOfReturn, setRateOfReturn] = useState(1);
+  const [delay, setDelay] = useState(1);
+  const [startToday, setStartToday] = useState();
+  const [delayedStart, setDelayedStart] = useState();
+  const [notionalLoss, setNotionalLoss] = useState();
+  const [err, setErr] = useState();
```

Comment:
can set false initially , although undefined is also a falsy value , but predefining as false , tells that setError is state which will have two values true and false